c-bindings-gen/src/main.rs

```diff
@@ -904,6 +932,7 @@ fn writeln_impl<W: std::io::Write>(w: &mut W, i: &syn::ItemImpl, types: &mut Typ
 						write!(w, "\n").unwrap();
 					} else if path_matches_nongeneric(&trait_path.1, &["From"]) {
 					} else if path_matches_nongeneric(&trait_path.1, &["Default"]) {
+						writeln!(w, "/// Creates a \"default\" {}. See other documentaiton for details on what this implies.", ident).unwrap();
```

Comment:
What is "other documentation" referencing?

Reply:
Indeed, you can see in the dif it was only used in config.rs.